include/git2/refspec.h

```diff
@@ -21,6 +21,23 @@
  */
 GIT_BEGIN_DECL
 
+/**
+ * Parse a given refspec string
+ *
+ * @param refspec a pointer to hold the refspec handle
+ * @param input the refspec string
+ * @param is_fetch is this a refspec for a fetch
+ * @return 0 if the refspec string could be parsed, 1 otherwise
```

Comment:
`1` otherwise is a bit weird. We always return `-1` for errors.

Reply:
Sorry the function is actually returning `-1`, but this is obviously a documentation error.